### ISSUE TITLE
The query window is unreadable in Dark theme

### ISSUE THREAD

**Issue by theelderwand** [microsoft/AzureStorageExplorer#626]
**Storage Explorer Version**: 1.4.1
**Platform**: Windows
**Architecture**: ia32
**Build Number**: 20180822.1
**Commit**: 14ea2643

#### Steps to Reproduce: ####
1. View -> Themes -> Dark
2. Navigate to a cosmos DB account and open any collection view
3. Click on filter and try to type in any filter string.

#### Expected Experience: ####
I expect the color combination to display a readable text

#### Actual Experience: ####
Unreadable text due to color mismatch, text shows up only when highlighted
<img width="381" alt="repro1" src="https://user-images.githubusercontent.com/2137463/44894736-2760bd00-aca6-11e8-8fe2-edbd086bfdf7.png">
<img width="531" alt="repro2" src="https://user-images.githubusercontent.com/2137463/44894737-2a5bad80-aca6-11e8-9389-0ac9171f3b90.PNG">

**Comment by LarsKemmann** [comment]
Just reported this same issue [here](https://github.com/Microsoft/AzureStorageExplorer/issues/649) by mistake since this issue didn't show up for me when searching for existing issues. Sorry about the duplicate, and please get this fixed.

**Comment by szq-trd** [comment]
Still an issue with latest version. Very irritating as I prefer dark mode. Please fix soon if possible.

I'm guessing just a simple change to the theme. When mouse over the filter input box either change the text colour to black/grey or change the background colour to black/grey.

Storage Explorer Version: 1.8.0
Platform: Windows
Architecture: ia32
Build Number: 20190501.8
Commit: 891a4c34

Thank you!

**Comment by MendelBak** [comment]
This is also rather frustrating for me. I thiink @szq-trd has the simplest solution.

**Comment by MendelBak** [comment]
#676

**Comment by vincez** [comment]
Any update? This should be a quick fix...

**Comment by marshallr12** [comment]
In v1.10.1, I can't see the text when it the query predicate text is selected & focused, rather than is not selected.

**Comment by jinglouMSFT** [comment]
@southpolesteve this really needs to be fixed.  it's impacting everyone using the dark theme, and the bug has been there since 2018.  Thanks.,

**Comment by soheilfarahani** [comment]
I'm having the same issue. Any update?
Thanks though for all other stuff.

**Comment by chriscardwell-iq** [comment]
the query predicate text box looks like this when the cursor is not over it:
![image](https://user-images.githubusercontent.com/8418896/73807829-6ba59600-4793-11ea-9f2e-35ff409a2793.png)

but as soon as you mouse over, it looks like this and the text isn't visible:
![image](https://user-images.githubusercontent.com/8418896/73807850-7c560c00-4793-11ea-9321-bb6418098587.png)

even though if you highlight the text, it's definitely there:
![image](https://user-images.githubusercontent.com/8418896/73807862-8841ce00-4793-11ea-8774-2056986c27a1.png)

any update on a fix?

**Comment by chriscardwell-iq** [comment]
any update on this one? I can't imagine it's a ton of effort and it renders the dark theme almost unusable because you can't see what you're typing a majority of the time.

**Comment by aliuy** [comment]
Azure Storage Explorer support for Azure Cosmos DB has been discontinued as of v1.24.0. We recommend users to use Azure Cosmos DB’s Data Explorer, which can be found in [Azure Portal](https://portal.azure.com/) or as a stand-alone version in http://cosmos.azure.com/ for the best Azure Cosmos DB experience. You can also find additional Azure Cosmos DB tooling here: [Azure Cosmos DB Tools](https://developer.azurecosmosdb.com/tools)
